docker/Dockerfile.amd

```diff
@@ -2,72 +2,129 @@
 # This Dockerfile contains AMD compilers
 ##############################################################
 
+ARG IMAGE="rocm/dev-ubuntu-22.04:5.4.2-complete"
 ARG arch="aomp"
 
-ARG ROCM_VERSION=5.4.2
```

Comment:
This seems easier to maintain it's the exact same but can easily switch version and much easier to use